ruby-challenges.rb

```diff
@@ -31,3 +49,16 @@
 # Expected output: 'LEARN is not a palindrome'
 is_palindrome3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
+
+def palindrome(string)
+    new_string = string.downcase
```

Comment:
Yes, wonderful approach to make everything is lowercase to perform the evaluation